apps/src/templates/studioHomepages/CourseCard.jsx

```diff
@@ -24,19 +24,22 @@ class CourseCard extends Component {
     const {title, description, link, isRtl} = this.props;
 
     return (
-      <a href={link} style={styles.card}>
+      <div style={styles.card}>
```

Comment:
Note this now means that only the button, not the whole card is clickable. Nested <a> tags is an anti-pattern and the storybook tests got very grumpy about it.